Ghidra/Debug/Framework-TraceModeling/src/main/java/ghidra/trace/util/WrappingFunctionIterator.java

```diff
@@ -32,7 +32,7 @@ public WrappingFunctionIterator(Iterator<? extends Function> it) {
 
 	public <T extends Function> WrappingFunctionIterator(Iterator<T> it,
 			Predicate<? super T> filter) {
-		this.it = Iterators.filter(it, filter);
+		this.it = Iterators.filter(it, filter::test);
```

Comment:
This doesn't make sense to me. It is already a predicate so why would you create a lambda to make another predicate using `Predicate::test` which is it's `FunctionalInterface` method. Just forward the existing predicate.

Reply:
Does not compile